app/scripts/controllers/serviceInstance.js

```diff
@@ -24,11 +28,28 @@ angular.module('openshiftConsole')
       ServiceInstancesService.deprovision($scope.serviceInstance);
     };
 
+    $scope.showEditDialog = function() {
+      $scope.editDialogShown = true;
+    };
+
+    $scope.showConfigValues = false;
+
+    $scope.toggleConfigValues = function() {
```

Comment:
nit: I have a small preference for calling them parameter values in the code since it's what they're called in the API.